src/GPS.h

```diff
@@ -44,6 +44,7 @@ class GPSClass {
   float latitude();
   float longitude();
   float speed(); // Speed over the ground in kph
+  float speedRAW(); // Speed over the ground in knots
```

Comment:
Hi @crami,

First of all, thank you so much for taking time to submit this pull request!

In the [Arduino_MKRENV](https://github.com/arduino-libraries/Arduino_MKRENV) library, we use the style of adding a parameter to select the unit type for the return value.

So, for example, here we could have something like:
```
GPS.speed(); // returns GPS (ground) speed in km/h.
GPS.speed(KPH); // same as ^

GPS.speed(KNOTS); // returns GPS (ground) speed in knots
```

What do you think about this approach?